__tests__/e2e/constants/field.ts

```diff
@@ -166,15 +166,15 @@ export const ALL_FIELDS: E2eFieldMetadata[] = [
   {
     title: 'Birthday',
     fieldType: BasicField.Date,
-    val: format(new Date(), DATE_INPUT_FORMAT),
+    val: format(new Date(952970366), DATE_INPUT_FORMAT),
```

Comment:
Is this date value `952970366` special?

Reply:
<img width="648" alt="image" src="https://user-images.githubusercontent.com/25571626/225508103-29deab59-c585-43a5-9322-b05c3524fce7.png">
:)

Reply:
ok no but on a serious note, I couldn't leave it as the current date because our Storage mode's individual response page also displays the date of submission which would also be the current date. So the `expect()` for finding the response to this question would certainly always succeed, regardless of whether the response to the question was shown.